src/hooks/__internal__/useFloating/useFloating.test.tsx

```diff
@@ -41,120 +41,100 @@ const MockComponent = ({
   );
 };
 
-describe("useFloating", () => {
-  it("invokes computePosition with proper arguments", () => {
-    const computePositionSpy = jest.spyOn(floatingUi, "computePosition");
+test("computePosition is called with the correct arguments when MockComponent is rendered", () => {
+  const computePositionSpy = jest.spyOn(floatingUi, "computePosition");
 
-    render(<MockComponent isOpen strategy="fixed" placement="top" />);
-    expect(computePositionSpy.mock.calls[0][0]).toBe(
-      screen.getByTestId("reference-element")
-    );
-    expect(computePositionSpy.mock.calls[0][1]).toBe(
-      screen.getByTestId("floating-element")
-    );
-    expect(computePositionSpy.mock.calls[0][2]).toMatchObject({
-      strategy: "fixed",
-      placement: "top",
-    });
+  render(<MockComponent isOpen strategy="fixed" placement="top" />);
+  expect(computePositionSpy.mock.calls[0][0]).toBe(
+    screen.getByTestId("reference-element")
+  );
+  expect(computePositionSpy.mock.calls[0][1]).toBe(
+    screen.getByTestId("floating-element")
+  );
```

Comment:
**suggestion(non-blocking)**: what do you think of this change to make the assertions more readable? Not essential though.

```suggestion
  expect(computePositionSpy).toHaveBeenNthCalledWith(1,
    screen.getByTestId("reference-element")
  );
  expect(computePositionSpy).toHaveBeenNthCalledWith(2,
    screen.getByTestId("floating-element")
  );
```

Reply:
For the record I'd tried to do it like this originally but it was erroring so I had to change it to the current version. Unless I'm missing something?